asterisk/agi/application/controllers/CallsController.php

```diff
@@ -574,6 +626,14 @@ public function addheadersAction()
                 $this->agi->setSIPHeader("X-Info-Friend", $friend->getRequestURI($exten));
                 $this->agi->setSIPHeader("X-Info-MaxCalls", 0);
             }
+            $retail = $endpoint->getRetailAccount();
+            if (!empty($retail)) {
```

Comment:
Due to $retail can only be an object or null, I would change this condition to ($retail) or (!is_null($retail))